l10nru.properties

```diff
@@ -10,14 +10,14 @@ defaults.mavenArtifactId=sonar-l10n-ru-plugin
 10.8.description=Support SonarQube 10.8 (SonarQube Core 24.12)
 10.8.date=2025-01-07
 10.8.sqs=[10.8,LATEST]
-10.8.sqcb=[24.12,LATEST]
+10.8.sqcb=[25.1,LATEST]
 10.8.downloadUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/download/v10.8/sonar-l10n-ru-plugin-10.8.jar
 10.8.changelogUrl=https://github.com/1c-syntax/sonar-l10n-ru/releases/tag/v10.8
 
 10.7.description=Support SonarQube 10.7
 10.7.date=2024-11-28
 10.7.sqs=10.7
-10.7.sqcb=[24.12,LATEST]
+10.7.sqcb=[24.12]
```

Comment:
@vilchik-elena this shouldn't be a range for one version. Just set it to `10.7.sqcb=24.12`